test/integration/simple_example/controls/gcloud.rb

```diff
@@ -12,12 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+project_id = attribute("project_id")
+region = attribute("region")
+cloud_router_name = attribute("cloud_router_name")
+
 control "gcloud" do
-  title "gcloud"
+  title "Gcloud checks: enabled compute api, and cloud router existence"
 
-  describe command("gcloud --project=#{attribute("project_id")} services list --enabled") do
+  describe command("gcloud --project=#{project_id} services list --enabled") do
```

Comment:
nit: I think this check is not necessary as the module is not responsible for enabling this api.